cypress/integration/Radio.spec.ts

```diff
@@ -0,0 +1,46 @@
+import * as h from '../helpers';
+
+const getRadio = () => {
+  return cy.get(`[type="radio"]`);
+};
+
+describe('Radio', () => {
+  before(() => {
+    h.stories.visit();
+  });
+  ['Default'].forEach(story => {
```

Comment:
No tests for alert and error?

Reply:
I might be wrong, but individual  radios don't have error or alert states, i think that's only for radio group?

Reply:
You are correct, haha sorry. Will wait until you get the RadioGroup cypress tests up.